Src/Plugins/Security/SEC101_001.HttpAuthorizationRequestHeaderValidator.cs

```diff
@@ -42,25 +43,26 @@ static HttpAuthorizationRequestHeaderValidator()
         }
 
         protected override IEnumerable<ValidationResult> IsValidStaticHelper(ref string matchedPattern,
-                                                                             Dictionary<string, string> groups)
+                                                                             Dictionary<string, FlexMatch> groups)
```

Comment:
>FlexMatch [](http://example.com/codeflow?start=96&length=9)

The essential change. Now, data flows into the validator that captures the specific offset of every group within the larger file. A validator can choose one of these in order to focus the primary region of interest.